src/geodata2.erl

```diff
@@ -5,7 +5,7 @@
 %% gen_server callbacks
 -export([init/1, handle_call/3, handle_cast/2]).
 %% API
--export([lookup/1, start/0, start_link/1, stop/0, get_env/2, id/1, is_ipv6_mmdb/0]).
```

Comment:
**DANGER:** The func `is_ipv6_mmdb/0` got removed and it's VITAL to exist!
It's used for boodah and dyno to know if they can lookup for IPv6 or not.

Reply:
Done.